ingest-app/src/cmr/ingest/services/subscriptions_helper.clj

```diff
@@ -164,38 +149,92 @@
                                  (email-subscription-processing-lookback)))
               subscription (add-updated-since raw-subscription time-constraint)
+              sub-meta (-> (:metadata subscription)
+                           (json/decode true))
               subscriber-id (get-in subscription [:extra-fields :subscriber-id])
+              sub-type (keyword (get sub-meta :Type))
+              query-string (:Query sub-meta)
               sub-id (get subscription :concept-id)
               coll-id (get-in subscription [:extra-fields :collection-concept-id])
-              query-string (-> (:metadata subscription)
-                               (json/decode true)
-                               :Query)
               query-params (create-query-params query-string)
-              search-by-revision (merge {:revision-date time-constraint
-                                         :collection-concept-id coll-id
-                                         :token (config/echo-system-token)}
+              search-by-revision (merge {:revision-date time-constraint}
+                                        (when (= sub-type :granule)
+                                          {:collection-concept-id coll-id})
+                                        {:token (config/echo-system-token)}
                                         query-params)
-              gran-refs (search-gran-refs-by-collection-id context search-by-revision sub-id)
-              subscriber-filtered-gran-refs (filter-gran-refs-by-subscriber-id context gran-refs subscriber-id)]]
-    [sub-id subscriber-filtered-gran-refs subscriber-id subscription]))
+              concept-refs (if (= sub-type :collection)
+                             (search-collection-refs context search-by-revision sub-id)
+                             (search-gran-refs-by-collection-id context search-by-revision sub-id))
+              subscriber-filtered-concept-refs (filter-concept-refs-by-subscriber-id context concept-refs subscriber-id)]]
+    [sub-id subscriber-filtered-concept-refs subscriber-id subscription]))
 
 (defn- ^:redef send-email
   "Wrapper for postal-core/send-message"
   [email-settings email-content]
   (postal-core/send-message email-settings email-content))
 
+(defmulti create-email-content
+  (fn [from-email-address to-email-address concept-ref-locations subscription]
+    (keyword (:Type (json/parse-string (:metadata subscription) true)))))
+
+(defmethod create-email-content :granule
+  [from-email-address to-email-address concept-ref-locations subscription]
+  (let [metadata (json/parse-string (:metadata subscription))
+        concept-id (get-in subscription [:extra-fields :collection-concept-id])
+        meta-query (get metadata "Query")
+        sub-start-time (:start-time subscription)
+        sub-end-time (:end-time subscription)]
+    {:from from-email-address
+     :to to-email-address
+     :subject "Email Subscription Notification"
```

Comment:
Should they have similar subjects? So maybe this could be "New Granules Match Your Subscription"? or the other could just be this as well

Reply:
yea I just left it as the ticket stated.  Going to go with "Email Subscription Notification" on both for now.  If we decide to change it in the future that can be another ticket.